force-app/main/default/classes/ODRIntegration.cls

```diff
@@ -1,6 +1,21 @@
 public with sharing class ODRIntegration {
     public ODRIntegration() { }
 
+    @AuraEnabled
+    public static Account fetchPatientInformation(String phn){
+        Account patient = new Account();
+        try {
+            patient = [SELECT Id, Name, Patient_Identifier__pc, PersonBirthdate
```

Comment:
@youngbeanie you have to call ODR for the patient info. Not all PHNs are on SF. eg., they might need to look up a patient from before MoH started using SF.